examples/simple_test.go

```diff
@@ -0,0 +1,42 @@
+package examples
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tetratelabs/wazero/wasm"
+	"github.com/tetratelabs/wazero/wasm/wazeroir"
+)
+
+// Test_Simple implements a basic function in go ("hello"), called by a function defined Web Assembly ("run"):
+func Test_Simple(t *testing.T) {
+	// Decode simple.wasm which was pre-compiled from this text format:
+	//	(module
+	//		(import "" "hello" (func $hello))
```

Comment:
I didn't know that imports with empty module name is valid 😄

Reply:
nutty right? it is all over the wasm spec tests!